web/packages/teleport/src/Discover/Database/DeployService/DeployService.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2023 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+
+import { ManualDeploy } from './ManualDeploy';
+import { AutoDeploy } from './AutoDeploy';
+
+export function DeployService() {
+  const [deployMethod, setDeployMethod] = useState<'manual' | 'auto'>('auto');
```

Comment:
`'manual' | 'auto'` is used a couple of times, should it be moved to its own type?

Reply:
sorry i wasn't sure where the other one was 😅